src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.VisualBasic.DesignTime.targets

```diff
@@ -38,8 +38,9 @@
                           VB;
                           Managed;
                           ClassDesigner;
-                          PreventAutomaticMyApplication;
                           SharedProjectReferences;" />
+
+      <ProjectCapability Include="EnableMyApplication" Condition="$([MSBuild]::VersionGreaterThanOrEquals('$(TargetFrameworkVersion)','5.0')) And '$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(UseWindowsForms)' == 'true'" />
```

Comment:
>VersionGreaterThanOrEquals('$(TargetFrameworkVersion)','5.0')) [](start = 79, length = 62)

I seem to remember that this is convention is changing. It might be good to check with Daniel Plaisted.

Reply:
cc: @dsplaisted :)

Reply:
Also, can we count on `UseWindowsForms` being set in a .NET 5.0 project?

Reply:
Yes, according to Klaus but it will be good to double check.

---
In reply to: [468775896](https://github.com/dotnet/project-system/pull/6486#discussion_r468775896) [](ancestors = 468775896)

Reply:
This condition looks correct.